docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -3211,7 +3225,9 @@ interactive session with the `exec` command.
     HTTP/1.1 200 OK
     Content-Type: application/vnd.docker.raw-stream
 
+    {% raw %}
```

Comment:
Yep, what @thaJeztah said. In Liquid you print out a variable using `{{ braces }}`